src/App.tsx

```diff
@@ -1,28 +1,45 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Avoid linter disabling

Reply:
I check and use linter disable in two files TodoItem and in fetchClient, because there are mistake with label and TS mistake with any